tests/core/templates/test_a_hash_map.h

```diff
@@ -235,9 +235,13 @@ TEST_CASE("[AHashMap] Insert, iterate and remove many elements") {
 TEST_CASE("[AHashMap] Insert, iterate and remove many strings") {
 	const int elem_max = 432;
 	AHashMap<String, String> map;
+
+	// To not print WARNING: Excessive collision count (NN), is the right hash function being used?
+	ERR_PRINT_OFF;
```

Comment:
CC @Nazarwadim - is this expected?

Reply:
My reasoning around turning off errors is that the test is adding consecutive numbers as strings and that is causing the hash function to "collision". Also the WARNING is only printed when `DEV_ENABLED` is defined.

https://github.com/godotengine/godot/blob/master/core/templates/a_hash_map.h#L187-L192
```cpp
#ifdef DEV_ENABLED
	if (unlikely(distance > 12)) {
		WARN_PRINT("Excessive collision count (" +
			itos(distance) + "), is the right hash function being used?");
	}
#endif
```